docs/usage/shoot-networking-filter.md

```diff
@@ -31,3 +31,49 @@ spec:
 ...
 ```
 
+### Ingress Filtering
+
+By default, the networking filter only filters egress traffic. However, if you enable blackholing, incoming traffic will also be blocked.
+You can enable blackholing on a per-shoot basis.
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: Shoot
+...
+spec:
+  extensions:
+    - type: shoot-networking-filter
+      egressFilter:
+        blackholingEnabled: true
+...
+```
+Ingress traffic can only be blocked, if the source IP address is preserved.
+In order to preserve the source IP address of a client when accessing a Service via a load balancer,
+you can use the "externalTrafficPolicy" setting in the Service specification. It may also require additional configuration of the load balancer
+depending on the infrastructure provider. Please check the documentation and capabilities of the infrastructure provider's load balancer to ensure that source IP preservation is supported.
```

Comment:
Here, I added a small section about the load balancer boundaries. However, it is quite general by now. I wonder, if I really should mention concrete configurations, were it won't work. Maybe, also include configuration examples for Services? @ScheererJ WDYT?